mnist_rnn/main.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import print_function
+import argparse
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.optim.lr_scheduler import StepLR
+
+
+class Net(nn.Module):
+    def __init__(self):
+        super(Net, self).__init__()
+        self.rnn = nn.LSTM(input_size=28,hidden_size=64, batch_first=True)
+        self.batchnorm = nn.BatchNorm1d(64)
+        self.dropout1 = nn.Dropout2d(0.25)
+        self.dropout2 = nn.Dropout2d(0.5)
+        self.fc1 = nn.Linear(64, 32)
+        self.fc2 = nn.Linear(32, 10)
+
+
+    def forward(self, input):
+        # Shape of input is (batch_size,1,28,28)
+        # converting shape of input to (batch_size, 28,28)
+        # as required by RNN when batch_first is set True
+        input = input.reshape(-1,28,28)
+        output, hidden = self.rnn(input)
+
+        # RNN output shape is (seq_len, batch, input_size)
+        # Get last output of RNN
+        output = output[:,-1,:]
+        output = self.batchnorm(output)
+        output = self.dropout1(output)
```

Comment:
Following changes made similar to pytorch MNIST example [here](https://github.com/pytorch/examples/blob/master/mnist/main.py#L27).